packages/phishing-controller/src/utils.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Hotlist,
+  PhishingListState,
+  PhishingStalelist,
+} from './PhishingController';
+/**
+ * Fetches current epoch time in seconds.
+ *
+ * @returns the Date.now() time in seconds instead of miliseconds. backend files rely on timestamps in seconds since epoch.
+ */
+export const fetchTimeNow = (): number => Math.round(Date.now() / 1000);
+
+/**
+ * Determines which diffs are applicable to the listState, then applies those diffs.
+ *
+ * @param listState - the stalelist or the existing liststate that diffs will be applied to.
+ * @param hotlistDiffs - the diffs to apply to the listState if valid.
+ * @returns the new list state
+ */
+export const applyDiffs = (
+  listState: PhishingListState | PhishingStalelist,
```

Comment:
Nit: We can simplify this type here. While we do pass the state in directly, this is still OK to TypeScript because structurally it satisfies the requirement of the `PhishingStalelist` type.

```suggestion
  listState: PhishingStalelist,
```